packages/git/src/browser/git-scm-provider.ts

```diff
@@ -415,14 +415,14 @@ export class GitScmProvider implements ScmProvider {
             fileText = `${paths.length} files`;
         }
         return new ConfirmDialog({
-            title: 'Discard changes',
+            title: nls.localize('theia/git/replaceConfirmDiscardTitle', 'Discard changes'),
```

Comment:
I believe that the following is already used, and can be re-used:

```suggestion
            title: nls.localize('vscode.git/package/command.clean', 'Discard changes'),
```